matrix-neoboard-widget/package.json

```diff
@@ -36,6 +36,7 @@
     "@types/react-i18next": "^8.1.0",
     "@types/react-redux": "^7.1.33",
     "babel-plugin-transform-import-meta": "^2.2.1",
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```

Comment:
This and the depcheck changes are due to this warning which now happens:

```


    One of your dependencies, babel-preset-react-app, is importing the
    "@babel/plugin-proposal-private-property-in-object" package without
    declaring it in its dependencies. This is currently working because
    "@babel/plugin-proposal-private-property-in-object" is already in your
    node_modules folder for unrelated reasons, but it may break at any time.

    babel-preset-react-app is part of the create-react-app project, which
    is not maintianed anymore. It is thus unlikely that this bug will
    ever be fixed. Add "@babel/plugin-proposal-private-property-in-object" to
    your devDependencies to work around this error. This will make this message
    go away.
```